modules/developer_manual/pages/_partials/app/advanced/security/idp_setup/example_config.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+$CONFIG = array (
+    // ...remaining configuration
+    'openid-connect' => [
+        'client-id' => 'ownCloud',
+        'client-secret' => 'ownCloud',
+        'loginButtonName' => 'node-oidc-provider',
```

Comment:
Maybe something more sexy ......

Reply:
could not resist, maybe `sexy-node-oidc-provider` :rofl:

Reply:
Why does code need to be sexy?

Reply:
It is the Text on the Button. It is User facing.

Usually i'd Put the Name of the idp or the system behind there.

Reply:
Login is a pretty apt name.